wiremock/mappings/041-ciag-induction-mapping.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Nice example data; especially considering we dont have real examples from CIAG yet. Good work 👍